tests/test_wave_03.py

```diff
@@ -55,12 +55,17 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
-
-@pytest.mark.skip()
+    assert INTRIGUE_3 in friends_unique_movies
+    assert HORROR_1 in friends_unique_movies
+    assert FANTASY_4 in friends_unique_movies
+    
+    
+    for count in count_friends_unique_movies(amandas_data).values():
+        assert count == 1
```

Comment:
Hm, I think I see what you are checking for here. I would have you consider what this function does that the `get_friends_unique_watched` doesn't already do? Refer to my suggestions below on line 101 of `party.py`